src/gui/searchengine/nova/engines/piratebay.py

```diff
@@ -27,113 +27,156 @@
 # ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
 # POSSIBILITY OF SUCH DAMAGE.
 
+try:
+    #python3
+    from html.parser import HTMLParser
+    from http.client import HTTPSConnection as https
```

Comment:
We know that this is for python2 so act accordingly. Not need to try...except
The same for the nova3 script.